pom.xml

```diff
@@ -100,7 +100,7 @@
         <version.error-prone-javac>9+181-r4173-1</version.error-prone-javac>
         <version.guava-beta-checker>1.0</version.guava-beta-checker>
         <version.immutables>2.8.8</version.immutables>
-        <version.jdk>1.8</version.jdk>
+        <version.jdk>1.11</version.jdk>
```

Comment:
Shouldn't it be: 
`<version.jdk>11</version.jdk>`

Reply:
I need a coffee.

Reply:
(And this also shows we should prioritize #4 or some other solution that allows us to verify most/all changes in this repo.)